Danki2/Assets/Scripts/Actor/Enemy/Enemy.cs

```diff
@@ -1,3 +1,9 @@
 ﻿public abstract class Enemy : Actor
 {
+    protected override void Start()
+    {
+        base.Start();
+
+        this.gameObject.tag = "Enemy";
```

Comment:
I don't think this should use a string literal here, it'd be better if this had access to a shared string somewhere so that it's all in one place. Same for Player.